unittests/tools/test_kubehunter_parser.py

```diff
@@ -43,10 +43,9 @@ def test_kubehunter_parser_empty_with_error(self):
             findings = parser.get_findings(testfile, Test())
             testfile.close()
 
-            self.assertTrue(
-                "KubeHunter report contains errors:" in str(context.exception)
-            )
-            self.assertTrue("ECONNREFUSED" in str(context.exception))
+        self.assertEqual(
```

Comment:
Did the contents of this exception change? Or was this maybe an error?

Reply:
In my opinion, this part was copy-pasted from different unittests and never properly tested. It hasn't been evaluated because it was skipped (because of `ValueError` error).
Check this: https://github.com/DefectDojo/django-DefectDojo/actions/runs/7711228961/job/21016345466
```
2024-01-30T12:50:40.0190551Z [36muwsgi_1         |[0m ======================================================================
2024-01-30T12:50:40.0191264Z [36muwsgi_1         |[0m FAIL: test_kubehunter_parser_empty_with_error (unittests.tools.test_kubehunter_parser.TestKubeHunterParser.test_kubehunter_parser_empty_with_error)
2024-01-30T12:50:40.0191529Z [36muwsgi_1         |[0m ----------------------------------------------------------------------
2024-01-30T12:50:40.0191738Z [36muwsgi_1         |[0m Traceback (most recent call last):
2024-01-30T12:50:40.0192422Z [36muwsgi_1         |[0m   File "/app/unittests/tools/test_kubehunter_parser.py", line 46, in test_kubehunter_parser_empty_with_error
2024-01-30T12:50:40.0192587Z [36muwsgi_1         |[0m     self.assertIn(
2024-01-30T12:50:40.0193131Z [36muwsgi_1         |[0m AssertionError: 'KubeHunter report contains errors:' not found in 'Expecting value: line 1 column 1 (char 0)'
2024-01-30T12:50:40.0193257Z [36muwsgi_1         |[0m 
```